ui/ui-frontend/projects/pastis/src/app/user-actions/save-profile/save-profile.component.ts

```diff
@@ -211,133 +197,83 @@ export class UserActionSaveProfileComponent implements OnInit, OnDestroy {
+          ...this.profileDescription,
+        };
+      } else {
+        this.subscriptions.add(
+          this.fileService.notice.subscribe((value: ProfileDescription) => {
+            this.profile = { ...value, ...profile };
+            this.profileDescription = value;
+          }),
+        );
+      }
+      this.subscriptions.add(this.saveArchiveProfile().subscribe());
+    }
   }
 
-  private attach(): Observable<any> {
-    return this.dialog
-      .open<SelectNoticeComponent, PastisDialogDataCreate>(SelectNoticeComponent, {
-        width: '800px',
-        panelClass: 'pastis-popup-modal-box',
-        data: {
-          height: 'auto',
-          isSlaveMode: this.isSlaveMode,
-          titleDialog: this.popupSaveSelectNoticeTitleDialog,
-          subTitleDialog: this.popupSaveCreateNoticeSubTitleDialog,
-          okLabel: this.popupSaveCreateNoticeOkLabel,
-          cancelLabel: this.popupSaveCreateNoticeCancelLabel,
-          profileType: this.profileService.profileType,
-          profileVersion: this.profileService.profileVersion,
-        },
-      })
-      .afterClosed()
-      .pipe(
-        filter((result) => result?.success),
-        switchMap((targetNoticeEvent) => {
-          const profileDescription = targetNoticeEvent.data;
-          const data: FileNode[] = this.data;
-          let action: Observable<any>;
-
-          if (this.profileService.isMode(ProfileType.PUA)) {
-            action = this.archiveUnitProfileSaverService.update(profileDescription, data);
-          }
+  private attach(attachementdata: any): Observable<any> {
+    const profileDescription = attachementdata;
+    const data: FileNode[] = this.data;
+    let action: Observable<any>;
 
-          if (this.profileService.isMode(ProfileType.PA)) {
-            const profile = this.noticeService.paNotice(profileDescription, targetNoticeEvent.profileVersion, false);
+    if (this.profileService.isMode(ProfileType.PUA)) {
+      action = this.archiveUnitProfileSaverService.update(profileDescription, data);
+    }
 
-            action = this.archiveProfileSaverService.attach(profile, profileDescription, data);
-          }
+    if (this.profileService.isMode(ProfileType.PA)) {
+      const profile = this.noticeService.paNotice(profileDescription, this.profileService.profileVersion, false);
+      action = this.archiveProfileSaverService.attach(profile, profileDescription, data);
+    }
 
-          return action;
-        }),
-        tap({
-          next: () => this.success('PROFILE.LIST_PROFILE.PROFILE_PREVIEW.MODIFICATION_SUCCESS'),
-          error: ({ error }: { error: VitamUIExceptionResponse }) => this.displayLogbookOperationSnackBar(error),
-        }),
-      );
+    return action.pipe(
+      tap({
+        next: () => this.success(this.translateService.instant('PROFILE.LIST_PROFILE.PROFILE_PREVIEW.MODIFICATION_SUCCESS')),
```

Comment:
Pas la peine, ça traduit déjà automatiquement.
```suggestion
        next: () => this.success('PROFILE.LIST_PROFILE.PROFILE_PREVIEW.MODIFICATION_SUCCESS'),
```

Reply:
Ça n'a pas été corrigé